app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -248,9 +248,13 @@ def construct_network_interfaces(network_interfaces_list, external_ip = nil)
 
     def check_google_key_path
       return if key_path.blank?
-      unless File.exist?(key_path)
+      if File.exist?(key_path)
+        !!JSON.parse(File.read(key_path).chomp)
```

Comment:
[fog-google](https://github.com/fog/fog-google/blob/c6b0533eec70451efc76fbf02470e6abfe24a734/lib/fog/google/shared.rb#L240) is checking `client_email` and `private_key` can we do something like:
```suggestion
        key_hash = JSON.parse(File.read(key_path).chomp)
        unless key_hash.key?('client_email') && key_hash.key?('private_key')
          errors.add#..
          # ....
```

Reply:
Updated PR with suggested changes.